core/tests/webdriverio_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -175,8 +177,9 @@ var TopicsAndSkillsDashboardPage = function() {
     await waitFor.visibilityOf(
       assignSkillToTopicButton,
       'Assign skill to topic buttons taking too long to appear');
+    await waitFor.numberOfElementsToBe(
```

Comment:
Also, numberOfElementsToBe doesn't seem to be a general case. Suppose in the future if we use it for tracking the change in a mathematical value, then `numberOfElementsToBe` will not be relatable to it.

Reply:
We just want to make sure the number of elements present in the DOM should be 1. Before we were expecting this to be 1 but here we are waiting till it becomes 1. And this function can be used for checking the number of elements present at that step, we can't use it for tracking the change in a mathematical value.